src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
 // write your code here
+
+const populationElements = document.querySelectorAll('.population');
+
+const populations = Array.from(populationElements).map((pop) => {
+  return Number(pop.textContent.split(',').join(''));
+});
+
+const totalPopulation = populations.reduce((sum, population) => {
+  return sum + population;
+}, 0);
```

Comment:
In your reduce function, you are using 'sum' and 'population' as parameter names. While these names are not incorrect, they could be more descriptive. Consider renaming 'sum' to 'total' and 'population' to 'currentPopulation'.